client/components/thank-you/index.tsx

```diff
@@ -195,9 +196,17 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 				href: CALYPSO_CONTACT,
 				title: translate( 'Ask a question' ),
 			},
+			...( documentationURL
+				? [
+						{
+							href: documentationURL,
+							title: translate( 'View plugin documentation' ),
+						},
+				  ]
+				: [] ),
```

Comment:
I think we can remove this as it is a duplicate of the section above?
![CleanShot 2022-04-27 at 10 55 11@2x](https://user-images.githubusercontent.com/12430020/165469655-e7394cee-0edf-4908-95d9-688f0bda400b.png)

Reply:
Fixed at https://github.com/Automattic/wp-calypso/pull/63051/commits/f36e15bdaa3c00f8d279d706c54d2b48b6b65817